.travis.yml

```diff
@@ -111,6 +111,52 @@ matrix:
         before_script: skip
         after_script: skip
 
+    - name: Robot Framework Tests
+      env:
+        - TEST_GROUP=acceptance
+      php: "7.4"
+      dist: xenial
+      install:
+        - chmod -R a+x config/Shared/ci/travis/
+        - composer install --optimize-autoloader --no-interaction
+        - config/Shared/ci/travis/install_elasticsearch.sh
+        - config/Shared/ci/travis/install_mailcatcher.sh
+        - >
+          echo "
+          # This file is required by python's package manager PIP
+          # Spryker uses this to run Robot tests in CI
+
+          robotframework==3.2.1
+          robotframework-seleniumlibrary==4.4.0
+          robotframework-seleniumtestability==1.1.0
+          " > requirements.txt
```

Comment:
requirements.txt is like composer.json.

There is another standard called Pipfile which would be better to use if these dependency files become part of the project.